graphix/extraction.py

```diff
@@ -77,7 +76,13 @@ def get_fusion_network_from_graph(
     list
         List of :class:`ResourceGraph` objects.
     """
-    use_rustworkx = isinstance(graph, RXGraphState)
+
+    try:
+        from graphix.graphsim.rxgraphstate import RXGraphState
+
+        use_rustworkx = isinstance(graph, RXGraphState)
+    except Exception:
```

Comment:
`ModuleNotFoundError` would be better, but I propose to let `GraphState` do `rustworkx` probing itself: https://github.com/thierry-martinez/graphix/commit/f7e503c6734a793f6c7247d95e6213b8213ba8d4